backend/controller/observability/observability.go

```diff
@@ -0,0 +1,207 @@
+package observability
+
+import (
+	"context"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+	"time"
+
+	"github.com/alecthomas/types/optional"
+
+	"github.com/TBD54566975/ftl/backend/controller/dal"
+	ftlv1 "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1"
+	"github.com/TBD54566975/ftl/backend/schema"
+	"github.com/TBD54566975/ftl/internal/log"
+	"github.com/TBD54566975/ftl/internal/model"
+)
+
+const name = "ftl.xyz/ftl/runner"
```

Comment:
I'm not sure this is correct. Many of these metrics can come from the controller if I'm reading this correctly. I also think `runner` specific metrics should be handled in `backend/runner/` packages probably.

Reply:
dropped and went with other naming feedback (e.g. `fsm.call`)

the xyz is a domain that I've seen used in conjunction with ftl. The domain/path was motivated by some of the example otel integration code I learned from while ramping. I personally prefer the naming scheme that you two have suggested here.